### DISCUSSION TITLE
just have a question

### DISCUSSION THREAD

**Discussion by ProjecttFrostt** [VolcanicArts/VRCOSC#55] [Q/A]
would u be open for a commission at all? or at least help me with something?

**Comment by VolcanicArts** [comment]
What did you have in mind?

**Reply by ProjecttFrostt** [reply]
Well Ive made a discord rpc with node js but i kinda want to make one with window application but don't really know how to and kinda want to make it detect like steam games im playing within and show in the details area but not sure if it's possible or not

**Reply by ProjecttFrostt** [reply]
> What did you have in mind?

U think you will be able to help?

**Reply by VolcanicArts** [reply]
There's a multitude of UI frameworks you can use for whatever language you're doing this in. Detecting games is done by checking what processes are open on Windows
